containers/ddev-webserver/Dockerfile

```diff
@@ -101,6 +101,9 @@ RUN phpdismod xdebug && curl -sSL --fail --output /usr/local/bin/phive "https://
 RUN set -o pipefail && curl -sSL https://github.com/pantheon-systems/terminus/releases/download/$(curl --silent "https://api.github.com/repos/pantheon-systems/terminus/releases/latest" | perl -nle'print $& while m{"tag_name": "\K.*?(?=")}g')/terminus.phar --output /usr/local/bin/terminus && chmod 777 /usr/local/bin/terminus
 RUN set -o pipefail && curl -sSL https://github.com/platformsh/platformsh-cli/releases/download/$(curl --silent "https://api.github.com/repos/platformsh/platformsh-cli/releases/latest" | perl -nle'print $& while m{"tag_name": "\K.*?(?=")}g')/platform.phar --output /usr/local/bin/platform && chmod 777 /usr/local/bin/platform
 
+RUN mkdir -p "/opt/phpstorm-coverage" && \
+    chmod a+rw "/opt/phpstorm-coverage"
```

Comment:
So you decided to let the coverage stuff be transient, not keep it in a volume?

Reply:
Yes, a volume would probably be the 100% solution, but as nobody else asked for the coverage feature till now, I would like to keep it simple. 
PHPStorm doesn't use older coverage reports by default anyway (No coverage indicatos available till you re run the test or you manually load reports).
So let's keep it simple. :)

Reply:
OK, thanks. I assume this PR is working for you with PhpStorm + coverage, so I'll pull it when you're ready.